litani

```diff
@@ -281,9 +172,39 @@ def get_args():
         flags = arg.pop("flags")
         graph_pars.add_argument(*flags, **arg)
 
+    get_jobs_pars = subs.add_parser("get-jobs", help="print jobs")
+    for arg in [{
+        "flags": ["-f", "--out-file"],
+        "help": "write the list of jobs to a JSON file, default: stdout",
+        "default": sys.stdout,
+        "metavar": "F",
+        "type": argparse.FileType("w")
+    }]:
+        flags = arg.pop("flags")
+        get_jobs_pars.add_argument(*flags, **arg)
+    get_jobs_pars.set_defaults(func=lib.jobs.print_jobs)
+
+    set_jobs_pars = subs.add_parser("set-jobs",
```

Comment:
Is indentation off?

Reply:
what makes you say that?

Reply:
Disregard earlier comment. Indentation's fine.